assets/js/admin-dashboard-report.js

```diff
@@ -0,0 +1,42 @@
+/* global $ */
+
+// admin user tools
+$(function () {
+  // auto-size the record content
+  var textarea = $('textarea')
+  textarea.height(textarea[0].scrollHeight)
+
+  // save
+  $('#save-btn').on('click', function () {
+    // TODO
+    var data = textarea.val()
+    $('#error-general').text('')
+    $.ajax(location.pathname, {method: 'post', contentType: 'application/json; charset=utf-8', dataType: 'json', data})
```

Comment:
This will need CSRF. Does it work right now? Does this branch not have CSRF in it? (If CSRF is implemented, but this form works, then that indicates a bug in our CSRF handling.)